### ISSUE TITLE
TS1192: Module '"[...]/node_modules/cem-plugin-better-lit-types/dist/index"' has no default export.

### ISSUE THREAD

**Issue by adaliszk** [Uscreen-video/cem-plugin-better-lit-types#1]
When using the plugin by:
```typescript
import extractLitTypes from 'cem-plugin-better-lit-types'
```

Typescript reports a problem that the package itself has no default export:
```
TS1192: Module '"[...]/node_modules/cem-plugin-better-lit-types/dist/index"' has no default export.
```

This is technically true, as the index does not have a default export, but the plugin is supposed to supersede that. While on runtime, this does happen; it is not detected in the linting stage.

**Comment by Bitaru** [comment]
Thanks! I haven't faced such an issue with the latest TS but seems like we have to change the way we export :)

**Comment by Bitaru** [comment]
@adaliszk just released `0.1.16` to npm. Will make a better version release flow next week.
Thank you for contributing!

**Comment by adaliszk** [comment]
Thank you for the quick fix!
